src/Pages/Product/Product.js

```diff
@@ -1,8 +1,211 @@
+                          <br />
+                          스윙 캡 라이언은 볼 캡형태의 가장 기본이 되는 라인으로
+                          <br />
+                          어디서든 활용이 가능합니다.
+                          <br />
+                          필드 위에서도, 일상에서도 어디든 잘 어울릴 수 있는
+                          아이템이에요.
+                          <br />
+                          라이언의 스윙과 감탄사가 로고로 플레이 된 게 특징이며,
+                          <br />
+                          화이트 배경의 화사한 느낌을 주는 스윙 캡을 소개합니다.
+                        </p>
+                      </li>
+                    </ul>
+                    {/* ~~product detail images*/}
+                  </div>
+                </div>
+                {/* product floating */}
+                <aside className="FloatingOptionOutbox">
+                  <div className="FloatingOptionInnerbox">
+                    {/* product floating header~~~~~~*/}
+                    <header className="productOptionHeading">
+                      <p className="ProductCategory">
+                        {productInformation.category}
+                      </p>
+                      <h1 className="productName">{productInformation.name}</h1>
+                      <p className="ProductPrice">{productInformation.price}</p>
+                      <ul>
+                        <li className="saveCash">
+                          <span className="title">적립캐시</span>
+                          <span className="content">
+                            회원 구매 시 3% 최종 금액에 따라 변동 가능
+                          </span>
+                        </li>
+                        <li className="deliveryFee">
+                          <span className="title">배송비</span>
+                          <span className="content">5만원 이상 무료배송</span>
+                        </li>
+                      </ul>
+                    </header>
+                    {/* ~~~~~~product floating header*/}
+                    {/* product floating Amount~~~~~*/}
+                    <div className="productQuantity">
+                      <span className="quantityText">수량</span>
+                      <span className="quantityGroup">
+                        <button className="minus">-</button>
+                        <input
+                          className="countNum"
+                          type="text"
+                          maxlength="2"
+                          value="1"
+                        />
+                        <button className="plus">+</button>
+                      </span>
+                    </div>
+                    {/* ~~~~product floating Amount*/}
+                    {/* product floating options~~~~*/}
+                    <div className="options">
+                      <ul>
+                        <li>
+                          <img
+                            class="optionImg optionOne"
+                            alt="상품 옵션1"
+                            src="http://localhost:3000/images/productDetail/wear/hat1.png"
```

Comment:
```suggestion
                            src="/images/productDetail/wear/hat1.png"
```
슬래시로 시작해도 같은 표현일 것 같네요!

Reply:
넵!